nucypher/characters/chaotic.py

```diff
@@ -0,0 +1,263 @@
+        return ciphertext
+
+
+class BobGonnaBob(Bob, DKGOmniscient):
+    """
+    A Bob that, if the proper knowledge lands in his hands, is all too happy to perform decryption without Ursula.
+
+    After all, Bob gonna Bob.
+    """
+
+    def __init__(self, session_seed=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _derive_dkg_parameters(
+        self, ritual_id: int, ursulas, ritual, threshold
+    ) -> DkgPublicParameters:
+        return self._dkg_insight.dkg.public_params
+
+    class DKGOmniscientDecryptionClient(ThresholdDecryptionClient):
+        def gather_encrypted_decryption_shares(
+            self,
+            encrypted_requests,
+            threshold: int,
+            timeout: float = 10,
+        ) -> Tuple[
+            Dict[ChecksumAddress, EncryptedThresholdDecryptionResponse],
+            Dict[ChecksumAddress, str],
+        ]:
+            # Set aside the power instance for use later, in the loop.
+            trdp = (
+                self._learner._dkg_insight.fake_ritual.threshold_request_decrypting_power
+            )
+            responses = {}
+
+            # We only really need one encrypted tdr.
+            etdr = list(encrypted_requests.values())[0]
+
+            for validator, validator_keypair in zip(
+                self._learner._dkg_insight.validators,
+                self._learner._dkg_insight.validator_keypairs,
+            ):
+                dkg = ferveo.Dkg(
+                    tau=self._learner._dkg_insight.tau,
+                    shares_num=self._learner._dkg_insight.shares_num,
+                    security_threshold=self._learner._dkg_insight.security_threshold,
+                    validators=self._learner._dkg_insight.validators,
+                    me=validator,
+                )
+
+                # We can also obtain the aggregated transcript from the side-channel (deserialize)
+                aggregate = ferveo.AggregatedTranscript(
+                    self._learner._dkg_insight.aggregation_messages
+                )
+                assert aggregate.verify(
+                    self._learner._dkg_insight.shares_num,
+                    self._learner._dkg_insight.aggregation_messages,
+                )
+
+                decrypted_encryption_request = trdp.decrypt_encrypted_request(etdr)
+                ciphertext = decrypted_encryption_request.ciphertext
+
+                # Presuming simple for now.  Is this OK?
+                decryption_share = aggregate.create_decryption_share_simple(
+                    dkg,
+                    ciphertext,
+                    self._learner._dkg_insight.conditions_bytes,
+                    validator_keypair,
+                )
+
+                decryption_share_bytes = bytes(decryption_share)
+
+                ##### Uncomment for sanity check
+                # ferveo.DecryptionShareSimple.from_bytes(decryption_share_bytes)  # No IOError!  Let's go!
```

Comment:
Why not make this an assertion?
```suggestion
                assert(ferveo.DecryptionShareSimple.from_bytes(decryption_share_bytes))   # No IOError!  Let's go
```